### PR TITLE
Update createmail.js

### PR DESCRIPTION
**Pull request by pAditya198**
Fixed issue #57 

### change the size of the logo and removed extra HTML tag.

### CODE REVIEW

**Comment by rossoskull** [comment]
Please share a screenshot.

File: createmail.js [hunk 1]
```diff
@@ -80,8 +14,8 @@ const createMail = function (username, email) {
         }
         
         .logo {
-            width: 200%;
-            height: 200%;
+            width: 30px;
```
Comment (Monal5031) [inline createmail.js]: Use `rem` instead, 30px is ~1.8rem and 60px = ~3.7

**Comment by pAditya198** [comment]
Screenshot (17)

**Comment by pAditya198** [comment]
the output of locally sen t mail

**Pull request merged**